brisk-reconciler.opam

```diff
@@ -1,19 +1,27 @@
 # This file is generated by dune, edit dune-project instead
 opam-version: "2.0"
-build: [
-  ["dune" "subst"] {pinned}
-  ["dune" "build" "-p" name "-j" jobs]
-  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
-  ["dune" "build" "-p" name "@doc"] {with-doc}
-]
-bug-reports: "https://github.com/brisk/brisk-reconciler/issues"
-homepage: "https://github.com/brisk/brisk-reconciler"
-license: "MIT"
 version: "1.0.0-alpha.0"
-dev-repo: "git+https://github.com/brisk/brisk-reconciler.git"
+license: "MIT"
+homepage: "https://github.com/brisk/brisk-reconciler"
+bug-reports: "https://github.com/brisk/brisk-reconciler/issues"
 depends: [
   "ocaml" {>= "4.4.0"}
   "dune" {>= "1.10"}
   "reason" {>= "3.4.0"}
   "ppxlib"
 ]
+build: [
+  ["dune" "subst"] {pinned}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
```

Comment:
Not sure what this is about. All I've done is tweak the dependencies, the dune-project `lang`, and run `esy`.

Reply:
New dune version 😄